src/operations.js

```diff
@@ -77,6 +63,17 @@ const compose2Optics = (optic1, optic2) => {
   return undefined
 }
 
+const upgradePrimitiveToOptic = (optic) => {
```

Comment:
I think you can add a unit test for this function to increase the coverage a bit 😉

Reply:
Done!